src/import/engine.rs

```diff
@@ -69,6 +70,17 @@ impl Engine {
 
         Ok(size)
     }
+
+    pub fn get_size_properties(&self) -> Result<SizeProperties> {
+        let mut res = SizeProperties::default();
+        let collection = self.get_properties_of_all_tables()?;
+        for (_, v) in &*collection {
```

Comment:
what's `&*`?

Reply:
Well, it dereferences the `collection` and then makes it iterable, that's just how things work, I'm not sure how to explain it clearly : (